### ISSUE TITLE
review scripts in package.json

### ISSUE THREAD

**Issue by tugorez** [cancunDevs/cancunDevs-back#3]
If i'm not wrong you could rewrite much of the scripts block to be 
```
{
  "lint": "eslint **/*.js;",
  "lint-fix-all": "lint --fix",
  "lint-fix": "eslint --fix",
}
```
instead of 
```
{
  "lint": "./node_modules/.bin/eslint **/*.js;",
  "lint-fix-all": "./node_modules/.bin/eslint --fix **/*.js;",
  "lint-fix": "./node_modules/.bin/eslint --fix"
}
```
https://github.com/cancunDevs/cancunDevs-back/blob/develop/package.json#L9

**Comment by yoniihdc** [comment]
As far as I know,`eslint **/*.js` is attempting to execute your global installation of eslint which is part of our `devDependencies`, I prefer to use the local installation to avoid issues if the user hasn't eslint installed.   
But let me check if we can use this packages as you mentioned and fix if it's needed.

**Comment by tugorez** [comment]
Not really, it is described [here](https://docs.npmjs.com/cli/run-script) the "find in path" and "exec" order which in this case is the local installation.

**Comment by yoniihdc** [comment]
Uff good to know that, then let me fix those scripts and I let you know when it's done. Thanks for the information 👍

**Comment by tugorez** [comment]
It's my pleasure @yoniihdc . Happy coding :)
